### ISSUE TITLE
📊Tracking: Amazon Managed Service for Grafana

### ISSUE THREAD

**Issue by ccfife** [aws/aws-cdk#14352]
Add your +1 👍  to help us prioritize high-level constructs for this service
--- 

### Overview:

Amazon Managed Service for Grafana (AMG) is a fully managed service that is developed together with Grafana Labs and based on open source Grafana. Enhanced with enterprise capabilities, AMG makes it easy for you to visualize and analyze your operational data at scale. Grafana is a popular open source analytics platform that enables you to query, visualize, alert on and understand your metrics no matter where they are stored.

[AWS Docs](https://docs.aws.amazon.com/grafana/latest/userguide/getting-started-with-AMG.html)  

### Maturity: Needs CloudFormation Support

See the [AWS Construct Library Module Lifecycle doc](https://github.com/aws/aws-cdk-rfcs/blob/master/text/0107-construct-library-module-lifecycle.md) for more information about maturity levels.

### Implementation:

See the [CDK API Reference](url) for more implementation details.

### Issue list:

--- 
This is a  📊Tracking Issue

**Comment by andreclaro** [comment]
Any update on this?

**Comment by peterwoodworth** [comment]
We are still lacking CloudFormation support for this. I recommend opening an issue in the [CloudFormation coverage roadmap](https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues) if you're interested - I currently don't see any issues about Grafana in there.

**Comment by youngjeong46** [comment]
I have requested a CloudFormation support for Managed Grafana [here](https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/1272). Please go +1 it for high visibility.

**Comment by elamaran11** [comment]
https://aws.amazon.com/about-aws/whats-new/2022/12/amazon-managed-grafana-support-aws-cloudformation/ 
CF Now supports AMG. Any updates on this.

**Comment by mengdic** [comment]
Hi team, AMG supports CDK L1 construct. See more info here: https://github.com/aws/aws-cdk/tree/main/packages/@aws-cdk/aws-grafana

**Comment by peterwoodworth** [comment]
Now that CloudFormation has support for Grafana, we are now able to start the process of designing and developing some L2 constructs for this service! To kick off the design process, we will need a willing contributor who can go through our [RFC process](https://github.com/aws/aws-cdk-rfcs) - start by submitting an issue in this repository if you think you can contribute 🙂

**Comment by youngjeong46** [comment]
@peterwoodworth just to confirm, we need to create a separate issue for the RFC process (outside of this one)? I'll take a look at the process in greater detail and take a stab at it.

**Comment by peterwoodworth** [comment]
@youngjeong46 Yes, exactly! Thanks a bunch for offering to take a look through it 🙂

**Comment by youngjeong46** [comment]
@peterwoodworth Submitted one - let me know if I'm missing anything https://github.com/aws/aws-cdk-rfcs/issues/471
